src/Conjure/UI/TypeScript.hs

```diff
@@ -1,153 +1,308 @@
 {-# LANGUAGE TemplateHaskell #-}
 {-# LANGUAGE KindSignatures #-}
 {-# OPTIONS_GHC -fno-warn-orphans #-}
+{-# LANGUAGE InstanceSigs #-}
```

Comment:
Remove conjure tsdef completely. And this module.

Reply:
We made a separate issue for this now.

Reply:
It broke again so I've removed it now